src/shared/codelens/typescriptCodeLensProvider.ts

```diff
@@ -84,48 +84,50 @@ export function initialize({
     }
 
     const command = getInvokeCmdKey('javascript')
-    context.subscriptions.push(vscode.commands.registerCommand(command, async (params: LambdaLocalInvokeParams) => {
-        const logger = getLogger()
+    context.subscriptions.push(
```

Comment:
This was a code formatter change. Hide whitespace changes to see a better delta.